client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -39,29 +76,27 @@ const CreateSite: FunctionComponent< {} > = () => {
 				<div className="create-site__content">
 					<div className="create-site__progress">
 						<div className="create-site__progress-steps">
-							{ progressSteps.map( ( step ) => (
-								<div key={ step } className="create-site__progress-step">
-									{ step }
-								</div>
-							) ) }
+							<div className="create-site__progress-step">{ steps.current[ currentStep ] }</div>
 						</div>
 					</div>
 					<div
 						className="create-site__progress-bar"
-						onAnimationEnd={ () => setCreateAndRedirect( true ) }
+						style={
+							{
+								'--progress': progress,
```

Comment:
Should the CSS var have some prefix since it'll live in global scope?

Reply:
It doesn't live in the global scope, it's scoped to the element (unless I misunderstand CSS custom properties scoping and inheritance).